packages/turf-ellipse/index.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="geojson" />
+
+type Feature = GeoJSON.Feature<any> | GeoJSON.GeometryObject;
+
+/**
+ * http://turfjs.org/docs/#ellipse
+ */
+declare function ellipse(feature1: Feature, feature2: Feature): boolean;
```

Comment:
... 🤔 I know Typescript is new for a lot of folks, but you could simply copy the Typescript definition from `@turf/circle` which will be very similar to the new ellipse module.

https://github.com/Turfjs/turf/blob/master/packages/turf-circle/index.d.ts